bin/nestjs-client-petstore.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
please also add an entry to 
https://github.com/OpenAPITools/openapi-generator/blob/3e5fb670e24c6a0239ba78737fa26538f5d9883a/bin/utils/ensure-up-to-date#L61

Reply:
Im sorry I don't understand this.